localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,29 +1,323 @@
- overlord_female:0 "Overlord"
-
- king_nightborne_male:0 "King"
- queen_nightborne_female:0 "Queen"
-
- grand_prince_nightborne_male:0 "Grand Prince"
- grand_princess_nightborne_female:0 "Grand Princess"
- grand_principality_nightborne:0 "Grand Principality"
-
- prince_nightborne_male:0 "Prince"
- princess_nightborne_female:0 "Princess"
- principality_nightborne:0 "Principality"
-
- lord_nightborne_male:0 "Lord"
- lady_nightborne_female:0 "Lady" 
\ No newline at end of file
+ ### Stromic ###
+ # Special
+ emperor_arathor_male:0 "Emperor"
+ emperor_arathor_female:0 "Empress"
+ empire_arathor:0 "Empire"
+
+ # Feudal
+ emperor_feudal_male_stromic:0 "Emperor"
+ emperor_feudal_female_stromic:0 "Empress"
+ empire_feudal_stromic:0 "Empire"
+
+ ### Kul'tiran ###
+ # Feudal
+ lord_kultiran_male_feudal:0 "Lord"
+ lady_kultiran_male_feudal:0 "Lady"
+ admiral_male_feudal:0 "Admiral"
+ admiral_female_feudal:0 "Admiral"
+ admiralty_feudal:0 "Admiralty"
+ grand_admiral_male_feudal:0 "Grand Admiral"
+ grand_admiral_female_feudal:0 "Grand Admiral"
+ grand_admiralty_feudal:0 "Grand Admiralty"
+
+ ### Zandalari ###
+ # Feudal
+ god_king_feudal:0 "God King"
+ god_queen_feudal:0 "God Queen"
+
+ prince_male_zandalari_kingdom:0 "Prince"
+ princess_female_zandalari_kingdom:0 "Princess"
+ prince_male_zandalari_empire:0 "Prince"
+ princess_female_zandalari_empire:0 "Princess"
+
+ ### Zandalari ###
+ # Feudal
+ frost_king_feudal:0 "Frost King"
+ frost_queen_feudal:0 "Frost Queen"
+
+ ### Dragons ###
+ dragonlord_male_dragon:0 "Dragon Lord"
+ dragonlady_female_dragon:0 "Dragon Lady"
+ lord_male_dragon:0 "Lord"
+ lady_female_dragon:0 "Lady"
+ overlord_male_dragon:0 "Overlord"
+ overlady_female_dragon:0 "Overlady"
+
+ prince_male_dragon_kingdom:0 "Prince"
+ princess_female_dragon_kingdom:0 "Princess"
+ prince_male_dragon_empire:0 "Prince"
+ princess_female_dragon_empire:0 "Princess"
+
+ ### Red Dragons ###
+ dragonking_male_red_dragon:0 "Dragon King"
+ dragonqueen_female_red_dragon:0 "Dragon Queen"
+ dragonprince_male_red_dragon:0 "Dragon Prince"
+ dragonprincess_female_red_dragon:0 "Dragon Princess"
+
+ ### Mantids ###
+ emperor_male_mantid:0 "Grand Emperor"
+ empress_female_mantid:0 "Grand Empress"
+
+ imperial_vizier_male_mantid:0 "Imperial Vizier"
+ imperial_vizier_female_mantid:0 "Imperial Vizier"
+ grand_vizier_male_mantid:0 "Grand Vizier"
+ grand_vizier_female_mantid:0 "Grand Vizier"
+ vizier_male_mantid:0 "Vizier"
+ vizier_female_mantid:0 "Vizier"
+
+ ### Qiraji ###
+ # Special
+ emperor_ahn_qiraj_male_qiraji:0 "Emperor"
+ empress_ahn_qiraj_female_qiraji:0 "Emperor"
+ empire_ahn_qiraj:0 "Empire"
+
+ # Others
+ count_male_qiraji:0 "Lord"
+ count_female_qiraji:0 "Lady"
+ county_qiraji:0 "Hive"
+ duke_male_qiraji:0 "High Overlord"
+ duke_female_qiraji:0 "High Overlady"
+ duchy_qiraji:0 "Brood"
+
+ ### Bronzebeard ###
+ # Feudal
+ count_feudal_male_bronzebeard:0 "Lord"
+ count_feudal_female_bronzebeard:1 "Lady"
+ county_feudal_bronzebeard:0 "Lordship"
```

Comment:
You missed 2 dwarven cultures: Frostborn and Old Dwarven.

Reply:
done in d4b7af7b2 and cf38a0de7